Development/DCM/main.py

```diff
@@ -25,19 +25,27 @@ def updateGUI(self):
         self.guiController.updateGUI()
 
     def loginButtonCB(self):
-        ret = self.accountController.signInUser(self.guiController.getLoginData())
-        print(ret)
-        # self.guiController.drawScreen(programmingScreen)
+        print(self.accountController.signInUser(self.guiController.getLoginData()).value)
+        #self.guiController.drawScreen(programmingScreen)
 
     def logoffButtonCB(self):
         self.guiController.drawScreen(loginScreen)
 
     def newUserButtonCB(self):
-        print("login button callback")
+        self.guiController.drawScreen(createUserScreen)
 
+    def createUserButtonCB(self):
+        x = self.accountController.makeNewUser(self.guiController.getNewUserData(),"hello")
```

Comment:
is hello the admin password? if so, please change it to "adminPassword" so that it's not just a random hello in the code. Probably best to just add it as a constant